module-4/backend/week_three.md

```diff
@@ -1,19 +1,73 @@
 ## Week Three - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. At a high level, what is Node?
+
+>Node is a JavaScript runtime built on Chrome's V8 engine.
+
 2. What is Express? What is Express similar to in the Ruby world?
+
+>Express is a lightweight framework to help organize your web application into an MVC architecture on the server side.
```

Comment:
Express isn't necessary for MVC and vice versa. Express is an un-opinionated framework that gives us tools to build APIs easier similar to Sinatra in Ruby